backend/src/tasks/pools-updater.ts

```diff
@@ -136,18 +136,18 @@ class PoolsUpdater {
       axiosOptions.httpsAgent = new SocksProxyAgent(socksOptions);
     }
 
-    while(retry < 5) {
+    while(retry < config.MEMPOOL.EXTERNAL_MAX_RETRY) {
       try {
-        const data = await axios.get(path, axiosOptions);
-        if (data.statusText !== 'OK' || !data.data) {
+        const data: AxiosResponse = await axios.get(path, axiosOptions);
+        if (data.statusText === 'error' || !data.data) {
```

Comment:
Changed this one since not every servers respond with `statusText === "OK"`

```js
{
  status: 200,
  statusText: '',
  headers: {
    'content-type': 'application/json',
    'transfer-encoding': 'chunked',
    date: 'Fri, 20 May 2022 13:54:11 GMT',
    connection: 'close'
  },
  config: {
    transitional: {
      silentJSONParsing: true,
      forcedJSONParsing: true,
      clarifyTimeoutError: false
    },
    adapter: [Function: httpAdapter],
    transformRequest: [ [Function: transformRequest] ],
    transformResponse: [ [Function: transformResponse] ],
    timeout: 30000,
    xsrfCookieName: 'XSRF-TOKEN',
    xsrfHeaderName: 'X-XSRF-TOKEN',
    maxContentLength: -1,
    maxBodyLength: -1,
    env: { FormData: [Function] },
    validateStatus: [Function: validateStatus],
    headers: {
      Accept: 'application/json, text/plain, */*',
      'User-Agent': 'mempool/v2.4.0-dev'
    },
    httpAgent: SocksProxyAgent {
      _events: [Object: null prototype] {},
      _eventsCount: 0,
      _maxListeners: undefined,
      timeout: null,
      maxFreeSockets: 1,
      maxSockets: 1,
      maxTotalSockets: Infinity,
      sockets: {},
      freeSockets: {},
      requests: {},
      options: {},
      shouldLookup: false,
      proxy: [Object],
      tlsConnectionOptions: {},
      promisifiedCallback: [Function: callback],
      [Symbol(kCapture)]: false
    },
    method: 'get',
    url: 'http://wizpriceje6q5tdrxkyiazsgu7irquiqjy2dptezqhrtu7l2qelqktid.onion/getAllMarketPrices',
    data: undefined
  },
}
```